File: 0448-find-all-numbers-disappeared-in-an-array/0448-find-all-numbers-disappeared-in-an-array.py
class Solution:
    def findDisappearedNumbers(self, nums: List[int]) -> List[int]:
        setNums = set(nums)
        arr = []

        for i in range(1, len(nums) + 1):
            if i not in setNums:
                arr.append(i)

        return arr

File: 0448-find-all-numbers-disappeared-in-an-array/0448-find-all-numbers-disappeared-in-an-array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var findDisappearedNumbers = function(nums) {
    const setNums = new Set(nums);
    const arr = [];

    for (let i = 1; i <= nums.length; i++) {
        if (!setNums.has(i)) {
            arr.push(i);
        }
    }
    return arr;
};